src/plugins/reconness.js

```diff
@@ -77,10 +77,11 @@ export default {
       }
     )
 
-    app.config.globalProperties.$agentStatus = Object.freeze(
+    app.config.globalProperties.$entityStatus = Object.freeze(
       {
         RUNNING: 1,
-        FINISHED: 2
+        FINISHED: 2,
+        WAITING: 3
```

Comment:
We dont need WAITING

Reply:
We decided to keep Waiting status to identified an agent that was executed and finished